ReactAndroid/src/main/java/com/facebook/react/views/image/events/ImageLoadEvent.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.views.image.events;
+
+import android.support.annotation.IntDef;
+
+import com.facebook.react.uimanager.events.Event;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+
+public class ImageLoadEvent extends Event<ImageLoadEvent> {
+  @IntDef({ON_ERROR, ON_LOAD, ON_LOAD_END, ON_LOAD_START, ON_PROGRESS})
+  @Retention(RetentionPolicy.SOURCE)
+  @interface ImageEventType {}
+
+  /**
```

Comment:
`/**` is for javadoc block. This is not javadoc, we normally use single line comment for such a thing. E.g:

    // Currently ON_ERROR and ON_PROGRESS are not implemented, these can be added
    // easily once support exists in fresco.